src/fw/drivers/sf32lb/stubs/ambient_light.c

```diff
@@ -0,0 +1,31 @@
+#include <inttypes.h>
```

Comment:
For stubs like these, can we put them in a non-board specific path? They're useful when bringing up other devices also.

Reply:
yep, will actually open a PR for that and take out stubs from here